samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/CandidateVotes.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Threading.Tasks;
+using NServiceBus;
+
+public class CandidateVotes :
```

Comment:
Fair point. Also for a reader would be easier to locate the saga code when it has the word "Saga" in the name.